lets-talk-template.fifthtry.site/FASTN.ftd

```diff
@@ -16,4 +16,4 @@ package: lets-auth.fifthtry.site
 
 -- fastn.app: Lets Talk App
 mount-point: /
-package: lets-talk.fifthtry.site
+package: lets-talk.fifthtry.site
```

Comment:
Changes to this file are unintentional (and we want to keep the newline in the end of file).